src/shared/config.ts

```diff
@@ -101,6 +164,56 @@ export const ConvictConfig = Convict<ServiceConfig>({
       format: 'Boolean',
       default: true
     }
+  },
+  SHARED: {
+    PEER_ENDPOINT: '0.0.0.0:4003',
+    ALS_ENDPOINT: '0.0.0.0:4002',
+    QUOTES_ENDPOINT: '0.0.0.0:3002',
+    TRANSFERS_ENDPOINT: '0.0.0.0:3000',
+    BULK_TRANSFERS_ENDPOINT: '',
+    DFSP_ID: 'dfsp_a',
+    JWS_SIGN: false,
+    JWS_SIGNING_KEY: '',
+    WSO2_AUTH: {
+      staticToken: '',
+      tokenEndpoint: '',
+      clientKey: '',
+      clientSecret: '',
+      refreshSeconds: 60
+    },
+    // Todo: Investigate proper key setup
+    TLS: {
+      inbound: {
+        mutualTLS: {
+          enabled: false
+        },
+        creds: {
+          ca: '',
+          cert: '',
+          key: ''
+        }
+      },
+      outbound: {
+        mutualTLS: {
+          enabled: false
+        },
+        creds: {
+          ca: '',
+          cert: '',
+          key: ''
+        }
+      },
+      test: {
```

Comment:
What's the purpose of `test` here?

Reply:
I'm not exactly sure. I just replicated https://github.com/mojaloop/sdk-scheme-adapter/blob/master/src/config.js 
The tls object is passed as is to the MojaloopRequest constructor. I can investigate an answer for you tomorrow (for me).

Reply:
Ah ok - yeah let's try and keep changes to the `sdk-standard-components` as light as possible, so we probably want to keep this the same for now.

Same goes for my other comment about type improvements.

Reply:
Ok. If we are putting improvements on the back burner this should be ready.